Nos/Views/AuthorCard.swift

```diff
@@ -76,7 +76,7 @@ struct AuthorCard: View {
                 }
                 .padding(.bottom, 12)
 
-                KnownFollowersView(author: author)
+                KnownFollowersView(source: currentUser.author, destination: author)
                     .padding(.top, -8)
```

Comment:
Are we able to remove this `-8` padding now? That was my understanding of [this comment](https://github.com/planetary-social/nos/pull/1140#discussion_r1610640462).

Reply:
Yes! I removed it and I think the resulting code is easier to read and reason about now.